runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryConfig.java

```diff
@@ -59,6 +59,26 @@ default boolean hideDuplicateBook()
 		description = "Whether to hide Varlamore Envoy, as it is only required in the Depths of Despair quest, and is never asked for."
 	)
 	default boolean hideVarlamoreEnvoy()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "hideDarkManuscript",
```

Comment:
This config is added twice for some reason--perhaps I rebased this wrong? Either way, remove one.

Reply:
Corrected to false (won't hide Dark Manuscripts) by default.